packages/inventory/src/containers/InventoryAddEdit/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,304 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`containers/InventoryAddEdit edits inventory item: form props 1`] = `
+Object {
```

Comment:
why not do an equality check for the props as opposed to having it as a snapshot, 
given that its automatically generated, I think it has minimal incentive to actually check why it changed,